.github/workflows/publish.yaml

```diff
@@ -8,6 +8,12 @@ name: Publish
   release:
     types:
       - created
+  # workflow_dispatch is just for testing, and it won't fully work.
+  workflow_dispatch:
+    inputs:
+      name:
+        description: fake version name
+        required: true
```

Comment:
Is this intended to stay in?

Reply:
I guess so. It doesn't do any harm to leave it here, and it'll help when we want to test the workflow in the future.